src/jit/gentree.cpp

```diff
@@ -1740,6 +1740,35 @@ bool GenTreeCall::IsHelperCall(Compiler* compiler, unsigned helper) const
     return IsHelperCall(compiler->eeFindHelper(helper));
 }
 
+void GenTreeCall::ReplaceCallOperand(GenTree** operandUseEdge, GenTree* replacement)
+{
+    assert(operandUseEdge != nullptr);
+    assert(replacement != nullptr);
+    assert(TryGetUse(*operandUseEdge, &operandUseEdge));
+
+    GenTree* originalOperand = *operandUseEdge;
+    *operandUseEdge = replacement;
+
+    const bool isArgument = (replacement != gtControlExpr) && ((gtCallType != CT_INDIRECT) ||
+        ((replacement != gtCallCookie) && (replacement != gtCallAddr)));
```

Comment:
I don't know if the formatting here is imposed by jit-format, but if so I would suggest breaking this into multiple statements, because the second operand to the first `&&` has been broken into two lines, which makes it hard to visually associate them correctly.

---
In reply to: [78809341](https://github.com/dotnet/coreclr/pull/7170#discussion_r78809341) [](ancestors = 78809341)